content/dstu2/authorization.md

```diff
@@ -2,195 +2,1061 @@
+workflow is a single token that is valid for 570 seconds (~10 minutes).
+For applications that need access to services for longer durations,
+Cerner's Ignite platform supports the concept of 
+["refresh tokens"](#refresh_tokens).
+
+With 'online_access', your application can continue to obtain access
+tokens on behalf of the user until:
+
+- The user explicitly logs out using their respective session 
+  authority.
+- The user's session expires or is administratively terminated.
+- Your application's access is terminated by the given
+  organization.
+  
+With 'offline_access', an application can continue to obtain
+access tokens in perpetuity on behalf of the user until:
+
+- The user or administrator explicitly terminates the original 
+  authorization grant.
+- The user's access to the resource(s) from the original authorization
+  grant is terminated.
+- Your application's access is terminated by the given
+  organization.
+
+##### Supported Scopes #####
+
+[Wildcard Scopes](http://docs.smarthealthit.org/authorization/scopes-and-launch-context/#wildcard-scopes) 
+are currently **not supported**.  An application is currently required to 
+specifically request each scope that it needs to run.
+
+Other combinations of scopes may be limited; please see the [FAQ](#faq) for
+known limitations.
+
+##### State ####
+
+When performing an authorization grant request, it is highly recommended
+that your client application establish a transient, one-time-use "state" value
+for each individual request you send.  If your application receives an 
+authorization response that does not include a value known to the current
+user's device, it should reject the response.  This mechanism is to protect your 
+application against "cross-site request forgery" classes of exploits.  For more
+information on these types of exploits, consult the RFC
+[OAuth 2.0 Threat Model and Security Considerations][OAUTH-THREAT].
+
+#### Audience ####
+
+When performing an authorization grant request, it is required your client
+application send the base URL of the FHIR<sup>®</sup> resource server for which 
+you will be sending access tokens to.  This is conveyed via an additional query 
+parameter in the grant request named "aud".  This parameter it utilized by Cerner's
+authorization server to protect your application against a form of exploit
+where an untrusted resource server advertises a legitimate authorization
+server in its conformance document.  Without this feature, a valid, authorized 
+access token could be inadvertently be sent by your application to the untrusted 
+party.
+
+Authorization grant requests that do not contain the audience parameter
+are automatically rejected by Cerner's authorization server.  The following
+is a fragment from a x-www-form-urlencoded grant request query string
+where the audience is provided:
+
+<pre class="terminal">
+&aud=https%3A%2F%2Fauthorization.cerner.com%2F
```

Comment:
This looks more like the url for the auth server and not the url for the FHIR server